doc/developers.md

```diff
@@ -258,6 +258,60 @@ But you don't need to worry about this too much.
 When you open a pull-request, codecov will report.
 And you can browse the [codecov TDMS pages online](https://codecov.io/gh/UCL/TDMS).
 
+### `MATLAB` Units {#matlab-unit-tests}
+
+The unit tests in the `tdms/tests/matlab` directory can be run with
+
+```bash
+matlab -batch run_tests
+```
+
+The `utils` folder is a collection of small MATLAB functions that allow us to edit the `data/pstd_input_file{2,3}D.m` files during the unit testing phase, without having to bloat the repository with an individual input file for each unit test.
+
+##### `test_iteratefdtd_matrix_source_combinations`
+
+The `iteratefdtd_matrix` function has a number of ways to produce the source terms from the information provided to it in an `input_file.m`.
+Some combinations of the variables that _can_ be provided however are incompatible - either not providing sufficient information to generate the source fields, or providing contradictory information.
+
+As such, the unit test `test_iteratefdtd_matrix_source_combinations` (provided as `arc_19` in [#208](https://github.com/UCL/TDMS/issues/208)) explicitly works through each possible combination of source-related inputs and checks for the corresponding error (or checks that no error is raised in the event the combination of inputs is valid).
+The various combinations and expected results are listed in the table below.
+
+`TD-field` is also know as `exi/eyi` in some docstrings.
```

Comment:
`````suggestion
To run the unit tests, go to the `tdms/tests/matlab` directory and use the command:

```bash
matlab -batch run_tests
```

In the `utils` folder, you'll find small MATLAB functions to edit the `data/pstd_input_file{2,3}D.m` files during the testing phase.

The `iteratefdtd_matrix` function has different ways to generate source terms from an input file.
However, some combinations of variables provided can be incompatible, resulting in errors or contradictory information.
The unit test `test_iteratefdtd_matrix_source_combinations` goes through each possible combination of source-related inputs, checking for errors or valid input combinations.
The combinations and expected results are listed in a table.

\note: `TD-field` is also known as `exi/eyi` in some docstrings.
`````
Please check the meaning (particularly the first sentence) is preserved.
(I'm not sure that `\note` works in Doxygen via markdown, you mind checking?
  it would work in a C++ comment string but there's not 100% support in md).

------
Although having said all of that... 

> `TD-field` is also know as `exi/eyi` in some docstrings.

... could we call it consistently?

Reply:
`\note` works (although don't use the `:`, since that gets taken for part of the text in the note).

> ... could we call it consistently

Maybe? I don't know how spread out through the codebase the contrast is. But I do know that comments (not just variables) switch between the two names/conventions frequently, so we can't hit the "refactor" button sadly :(